consent-ui/src/index.html

```diff
@@ -1,13 +1,13 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
-  <title>ConsentUi</title>
-  <base href="/">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <link rel="icon" type="image/x-icon" href="favicon.ico">
-</head>
-<body>
-  <consent-app-root></consent-app-root>
-</body>
+  <head>
+    <meta charset="utf-8" />
+    <title>ConsentUi</title>
+    <base href="/" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <link rel="icon" type="image/x-icon" href="./assets/icons/icons8-approval.png" />
```

Comment:
This icon should have clear name `favicon` in the resources as it would not be used anywhere else. You need to create new asset